microsoft/botframework-solutions
# Fixes for cross platform compilation (Visual Studio on Windows and OsX)

## Description
Fixed literal usage on Path.Combine to allow for paths on OSX. 
Replaced xpath usage on prebuild commands to use MSBuild copy commands.

## Related Issue
[https://github.com/Microsoft/AI/issues/376](https://github.com/Microsoft/AI/issues/376)

## Testing Steps
Compile using Visual Stuio on both OSX and Windows. Check the skills can be accessed correctly at run time. i.e asking "add milk to my shopping list" will work on both platforms (currently OSX will fail).

## Checklist
- [X ] I have commented my code, particularly in hard-to-understand areas
- [X ] I have added or updated the appropriate unit tests
- [X ] I have updated related documentation

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
